src/components/DropdownMenu/DropdownMenu.test.tsx

```diff
@@ -133,15 +134,60 @@ describe('<DropdownMenu />', () => {
 
   describe('@radix-ui prop forwarding', () => {
     test('should pass content props to Radix Content component', () => {
-      render(<DropdownMenu {...DEFAULT_PROPS} alignMenu="start" side="top" />);
-
+      render(<Dropdown.DropdownMenu {...DEFAULT_PROPS_MENU} alignMenu="start" side="top" />);
       expect(mockRadixContent).toBeCalledWith(expect.objectContaining({ align: 'start', side: 'top' }));
     });
 
     test('should pass root props to Radix Root component', () => {
-      render(<DropdownMenu {...DEFAULT_PROPS} open={true} defaultOpen={true} />);
-
+      render(<Dropdown.DropdownMenu {...DEFAULT_PROPS_MENU} open={true} defaultOpen={true} />);
       expect(mockRadixRoot).toBeCalledWith(expect.objectContaining({ open: true, defaultOpen: true }));
     });
   });
+
+  describe('menu header component', () => {
+    test('should render Header component', () => {
+      const { container } = render(
+        <Dropdown.DropdownMenu
+          {...DEFAULT_PROPS_MENU}
+          open={true}
+          defaultOpen={true}
+          header={<Dropdown.Header {...DEFAULT_PROPS_HEADER} className={mockClassName} item={mockText[0]} />}
+        />
+      );
+      expect(container.getElementsByClassName(mockClassName).length).toBe(1);
+      expect(screen.getByText(mockText[0])).toBeInTheDocument();
```

Comment:
Can we separate the `className` test, and just do the `toBeInDocument` one here? Just so we're testing one thing at a time.